examples/multiple-choice/index.html

```diff
@@ -0,0 +1,55 @@
+<title>&lt;multiple-choice&gt;</title>
+
+<link rel=stylesheet href=https://devpunks.herokuapp.com/typography.css></script>
+
+<script src=/snuggsi.es></script>
+<script name=polyfill src=//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.0.0/webcomponents-hi-ce.js></script>
+
+<!-- import works inline body -->
+<link rel=import href=/examples/header-group>
+
+<header-group>
+  <h2 slot=header>&lt;multiple-choice&gt;</h2>
+
+  <a  slot=subheader title='See more component examples' href=./>See more component examples</a>
+</header-group>
+
+<main>
+
+<multiple-choice>
+  <blockquote>{question}</blockquote>
+
+  <fieldset>
+    <legend>Answers</legend>
+
+    <ul>
+    <template name=answers>
+      <li>
+        <label>
+          <input name=answer type=radio value={#}> {self}
+        </label>
+    </template>
+  </ul>
+
+  </fieldset>
+
+  <footer>
+    <button onclick=onprevious>👈 Previous</button>
+    <button onclick=onnext>Next 👉</button>
+  </footer>
+
+</multiple-choice>
+
+</main>
+
+<script src=/examples/multiple-choice/index.es></script>
```

Comment:
i'd like to discuss a bit more about the 'why' behind the `index.format` naming convention, unless it's arbitrary. generally the reason to use it is when the index.* name can be inferred by default based on the path, e.g. `<script src=/example/multiple-choice/></script>` otherwise I think i'd think about using some kind of components-oriented file naming convention for development ease such as `multiple-choice/template.html`, `multiple-choice/definition.es`, `multiple-choice/style.css`thinking in terms of how i can easily find and open these files from an editor by component name, and each file name is at least slightly self-descriptive in terms of role relative to a component.